src/syscall_sdl.c

```diff
@@ -218,10 +218,10 @@ void syscall_draw_frame(riscv_t *rv)
 {
     state_t *s = rv_userdata(rv); /* access userdata */
 
-    /* draw_frame(screen, width, height) */
+    /* draw_frame(base, width, height) */
     const uint32_t screen = rv_get_reg(rv, rv_reg_a0);
-    const uint32_t width = rv_get_reg(rv, rv_reg_a1);
-    const uint32_t height = rv_get_reg(rv, rv_reg_a2);
+    const int32_t width = rv_get_reg(rv, rv_reg_a1);
```

Comment:
Let's follow the same type of the parameters passed to [SDL_CreateTexture](https://wiki.libsdl.org/SDL2/SDL_CreateTexture). That is, `int w, int h`.

Reply:
The function prototype `check_sdl` should be adjusted accordingly.